docs/user-guide/commands/iter8_gen.md

```diff
@@ -11,11 +11,13 @@ Generate experiment.yaml file by combining an experiment chart with values
 ### Synopsis
 
 
-Generate an experiment.yaml file by combining an experiment chart with values.
+Generate an experiment.yaml file by combining the Iter8 experiment chart with values.
 
-    $ iter8 gen --sourceDir /path/to/load-test-http --set url=https://httpbin.org/get
+	iter8 gen --set "tasks={http}" --set http.url=https://httpbin.org/get
 
-This command is intended for development and testing of experiment charts. For production usage, the launch command is recommended.
+This command supports setting values using the same mechanisms as in Helm. Please see  https://helm.sh/docs/chart_template_guide/values_files/ for more detailed descriptions. In particular, this command supports the --set, --set-file, --set-string, and -f (--values) options all of which have the same behavior as in Helm.	
```

Comment:
```suggestion
This command supports setting values using the same mechanisms as in Helm. Please see [https://helm.sh/docs/chart_template_guide/values_files/](https://helm.sh/docs/chart_template_guide/values_files/) for more detailed descriptions. In particular, this command supports the `--set`, `--set-file`, `--set-string`, and `-f` (`--values`) options all of which have the same behavior as in Helm.	
```

Reply:
iter8 command documentation is auto generated (using `iter8 gen`). So, we will need to fix this not directly in this repo but in the iter8 repo.